### PR TITLE
f-user-message@v1.2.0 - user message component tests

### PR DESCRIPTION
**Pull request by siggerzz**
v1.2.0
------------------------------
*February 9, 2021*

### Added
- Added `f-user-message.component.spec` tests
- Added accessibility tests

### CODE REVIEW

File: packages/components/molecules/f-user-message/src/components/UserMessage.vue [hunk 1]
```diff
@@ -3,13 +3,16 @@
...
         class="l-container">
         <div class="l-row">
             <div
                 :class="$style['c-userMessage-container']"
                 class="l-container--inner">
                 <clock-icon />
-                <p :class="$style['c-userMessage-text']">
+                <p
+                    :class="$style['c-userMessage-text']"
+                    data-test-id="content">
```
Comment (xander-marjoram) [inline packages/components/molecules/f-user-message/src/components/UserMessage.vue]:
```suggestion
                    data-test-id="user-message-content">
```

File: packages/components/molecules/f-user-message/test-utils/component-objects/f-user-message-selectors.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const USER_MESSAGE_COMPONENT = '[data-test-id="user-message-component"]';
+export const USER_MESSAGE_CONTENT = '[data-test-id="content"]';
```
Comment (xander-marjoram) [inline packages/components/molecules/f-user-message/test-utils/component-objects/f-user-message-selectors.js]:
```suggestion
export const USER_MESSAGE_CONTENT = '[data-test-id="user-message-content"]';
```

File: packages/components/molecules/f-user-message/test/specs/accessibility/axe-accessibility.spec.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
+const UserMessage = require ('../../../test-utils/component-objects/f-user-message.component');
```
Comment (xander-marjoram) [inline packages/components/molecules/f-user-message/test/specs/accessibility/axe-accessibility.spec.js]: Have you run the linter? Not sure it'll like the space after `require`

Comment (JoshuaNg2332) [reply packages/components/molecules/f-user-message/test/specs/accessibility/axe-accessibility.spec.js]: I'll fix this now.

**Review by xander-marjoram — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**